delivery_line_sale_line_position/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2021 Camptocamp SA
```

Comment:
```suggestion
# Copyright 2023 Camptocamp SA
```

Reply:
the year should always be the 1st one. If not specified the "end" is always "today".

Reply:
Thanks @simahawk 
I didn't know it